### PR TITLE
Hide sensitive Wechat configuration + remove default fields

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

File: config/notifiers.go [hunk 1]
```diff
@@ -101,14 +101,9 @@ var (
 	// DefaultWechatConfig defines default values for wechat configurations.
 	DefaultWechatConfig = WechatConfig{
 		NotifierConfig: NotifierConfig{
-			VSendResolved: true,
+			VSendResolved: false,
 		},
-		Message:   `{{ template "wechat.default.message" . }}`,
-		APISecret: `{{ template "wechat.default.api_secret" . }}`,
-		ToUser:    `{{ template "wechat.default.to_user" . }}`,
```
Comment (brian-brazil) [inline config/notifiers.go]: Why aren't these templated? We do for other notifiers.

Comment (simonpasquier) [reply config/notifiers.go]: I must admit I have no clue. @songjiayang any insight?

Comment (songjiayang) [reply config/notifiers.go]: Hi, guys, I don't known the benefit  of templated this fields, I support to delete it  like @simonpasquier pr.

Comment (stuartnelson3) [reply config/notifiers.go]: I say we leave it in and just update the wechat secrets to the `Secret` type for this PR.

Comment (simonpasquier) [reply config/notifiers.go]: Ack @stuartnelson3.  I've rebased the PR to remote the commit on the templated fields.

**Comment by berlinsaint** [comment]
@stuartnelson3  sorry about didn't solve the issues theses days because of some individual things.
and @songjiayang  thanks you help me solve the issue. 感谢！

**Pull request merged**